feature/default_sets.go

```diff
@@ -87,4 +87,8 @@ var defaultSetDesc = setDesc{
 		SetInit:    {}, // I
 		SetNodeAnn: {}, // N
 	},
+	lnwire.ProvideStorageOptional: {
+		SetInit:    {}, // I
+		SetNodeAnn: {}, // N
+	},
```

Comment:
we should add the ability to turn this on/off. In fact, I think we might initially want this off by default.

Reply:
Should we add a new lnd config for this, to indicate if the operator wants in on storing other peer's storage?

Reply:
yup. See `--protocol.zero-conf` for an example